src/modules/job-manager/prioritize.c

```diff
@@ -19,6 +19,7 @@
 
 #include "src/common/libjob/idf58.h"
 #include "src/common/libczmqcontainers/czmq_containers.h"
+#include <inttypes.h>
```

Comment:
Just for my own curiosity, why the added include here?

Reply:
Possibly vestigial, I was using PRId64 to get some debug prints and it got flagged.  The file does use int64_t though, which makes me think it might actually be correct so I left it.